accel/tcg/cputlb.c

```diff
@@ -2219,9 +2219,20 @@ uint64_t cpu_ldq_le_data_ra(CPUArchState *env, target_ulong ptr,
  * TODO(am2419): Ugly hack to avoid logging memory accesses that load capability
  * components as normal memory accesses. The caller is responsible for logging.
  */
-uint64_t cpu_ldq_cap_data_ra(CPUArchState *env, target_ulong ptr,
-                             uintptr_t retaddr)
-{
+target_ulong cpu_ld_cap_word_ra(CPUArchState *env, target_ulong ptr,
+                                uintptr_t retaddr)
+{
+    FullLoadHelper *full_load;
+    MemOp op;
+#if TARGET_LONG_BITS == 32
+    op = MO_TEUW;
+    full_load = MO_TE == MO_LE ? helper_le_lduw_mmu : helper_be_lduw_mmu;
```

Comment:
It'd be nice if upstream had MO_TL and there were a helper_te_ldtl_mmu or equivalent... surprised it's not useful for other things, though interestingly gdbstub.h defines a local ldtul_p. That, and a sttl_p, would have at least helped op_helper_cheri_common.c

Reply:
Maybe worth moving the ifdefs here to a header and using them here? Would be more readable IMO. I feel like there might be more cases where we can use MO_TL.